lib/screens/v2/candidate_generator/pre_fare.ex

```diff
@@ -20,16 +20,27 @@ defmodule Screens.V2.CandidateGenerator.PreFare do
             ],
             screen_takeover_left: [
               :full_screen_left
-            ]
+            ],
+            body_takeover_left: [:full_body_left]
           }},
          {:right,
           %{
             screen_normal_right: [
               :header_right,
               {:body,
```

Comment:
If you agree with the above suggestion, this should probably be renamed to `:body_right`, as well as the branch label `body_normal` -> `body_normal_right`.

(I wonder if there's a less cumbersome way to do this than appending `_left` and `_right` all over the place :thinking:)

Reply:
I've thought about that a bunch and can't think of a better way given the direction we went with both screens using the same URL. If each screen had its own URL, we wouldn't need `_left` or `_right` but would be a big change in the frontend.

Reply:
🤔  Why are the appends needed?  

I see why `:main_content_left` and `:main_content_right` need to be differentiated because they map to different components, but not so sure about the containers, like `:body_left` and `:screen_takeover_right`.  

As I've interpreted it so far, the containers are kinda just branches that don't need to keep track of whether they are on the right screen or left screen.  It's redundant to say `:body_left` because we've already gone down the `screen_normal_left` branch.  Just `:body` could work.  Could you explain what I'm missing?

Reply:
That's true Hannah—it's possible that only the "leaf node" slots (that actually get widgets dropped into them) need to have unique names across the whole template, since those names are used by the widgets themselves to indicate exactly which slot(s) they can be placed in.

I can try testing this out after the branch is merged! Don't want to burden Christian with the extra work.